File: app/src/main/java/foocafe/org/foocafe/SignUpCredential.java
package foocafe.org.foocafe;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class SignUpCredential implements Serializable {
    @SerializedName("user_id")
    String userID;

    @SerializedName("event_id")
    String eventID;

    public SignUpCredential(String userID, String eventID){
        this.userID = userID;
        this.eventID = eventID;
    }

    public String getUserID() {
        return userID;
    }

    public String getEventID() { return eventID; }

}

File: app/src/main/java/foocafe/org/foocafe/entities/SignUpCredential.kt
package foocafe.org.foocafe.entities

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class SignUpCredential(userID: String, eventID: String) : Serializable {
    @SerializedName("user_id")
    var userID: String
        internal set

    @SerializedName("event_id")
    var eventID: String
        internal set

    init {
        this.userID = userID
        this.eventID = eventID
    }
}